proto/p4/v1/p4data.proto

```diff
@@ -34,6 +34,30 @@ message P4Data {
     bytes enum_value = 12;  // serializable enums only
   }
 }
+message GenericData {
+  oneof data {
+    bytes bitstring = 1;
+    GenericVarbit varbitstring = 2;
+    float float = 3;
```

Comment:
This is a 32-bit float. If intended, it would be best to call it `float32`.  It might be better to use `float64` in a serialization spec for flexibility.

Reply:
Protobuf defines `float` as 32 bits and `double` as 64 bits, see https://protobuf.dev/programming-guides/encoding/#structure
We can just make up a new primitive name like `float32`, right?